tests/test_pipeline_raw_data.py

```diff
@@ -0,0 +1,83 @@
+"""
+This script checks the download of raw data in oemof-B3 and target rules of the snakemake pipeline
+that create empty time series and scalars in the directory 'raw'.
+"""
+import os
+import snakemake
+import shutil
+from oemof_b3.tools.testing_pipeline import get_repo_path, pipeline_output_test
+from oemof_b3.config import config
+
+logger = config.add_snake_logger("data_processing")
+
+# Delete data from test run of pipeline if True otherwise False
+delete_switch = True
```

Comment:
@Alaadin17 please keep the delete_switch in your branch for the case developer wants to keep all test results.